jax/experimental/array.py

```diff
@@ -401,3 +402,5 @@ def _array_local_result_handler(aval, sharding, indices):
   return lambda bufs: Array(aval, sharding, bufs, committed=True)
 pxla.local_result_handlers[(core.ShapedArray, pxla.OutputType.Array)] = _array_local_result_handler
 pxla.local_result_handlers[(core.ConcreteArray, pxla.OutputType.Array)] = _array_local_result_handler
+
+ndarray.register(array)
```

Comment:
`Array` the class? 

nit: Also add a new line at the bottom?

Reply:
How did the tests not fail?

Reply:
Ohh.. the tests did fail!

Reply:
Fixed